### ISSUE TITLE
Add Hide button for apps

### ISSUE THREAD

**Issue by Anti-Apple4life** [cockpit-project/cockpit#17868]
## Page: Apps, with some impact on overall layout as well

Add a button next to Delete button on the Apps page to hide the app from layout without entirely uninstalling it.
### Mockup:
![Screen Shot 2022-11-02 at 13 24 58](https://user-images.githubusercontent.com/59315931/199559182-f73bb25c-e18d-4f63-97e8-89ed244784d8.png)

**Comment by jelly** [comment]
What is the rationale behind this feature request? Is the idea to hide the app from the sidebar or just from the `Apps` page? And if hidden how would it be shown again?

**Comment by Anti-Apple4life** [comment]
The idea would be to hide the app from the sidebar

**Comment by marusak** [comment]
Why not just removing it when you want to hide it?

**Comment by Anti-Apple4life** [comment]
There is a bug on Arch when trying to remove built-in modules like Kernel Dump. It deletes the entire package.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by jelly** [comment]
Right, the Arch package has not split up kdump support. Ideally it would not show the page as it's unsupported. Please make a ticket in the Arch Linux bug tracker.

**Comment by garrett** [comment]
Renamed appropriately.

When there's a bug filed at the Arch Linux issue tracker, please post a link here. Thanks!

**Comment by martinpitt** [comment]
As far as I understand https://gitlab.archlinux.org/archlinux/packaging/packages/cockpit/-/blob/main/PKGBUILD?ref_type=heads#L79 , Arch stopped packaging the "Apps" page? @jelly ?

**Comment by jelly** [comment]
> As far as I understand https://gitlab.archlinux.org/archlinux/packaging/packages/cockpit/-/blob/main/PKGBUILD?ref_type=heads#L79 , Arch stopped packaging the "Apps" page? [@jelly](https://github.com/jelly) ?

Yup, that seems correct. Don't know what the reasoning is.

**Comment by martinpitt** [comment]
Thanks for confirming.
